src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseDataFileServiceTest.cs

```diff
@@ -1275,50 +1252,49 @@ public async Task GetInfo_MatchingSourceZipBlob()
         {
             var release = new Release();
 
-            var zipFile = new File
+            var zipReleaseFile = new ReleaseFile
```

Comment:
This is wrong as ReleaseFile links don't get created for files of type `DataZip` so leave the `zipFile` as a `File` here